### PR TITLE
Add username to register.htm if loginAttribute is username.

### PR DESCRIPTION
**Pull request by JBonnyDev**
Checks whether the login attribute is "username" and adds the required input field to register.htm. 
Issue reference: #284

### CODE REVIEW

File: components/account/register.htm [hunk 1]
```diff
@@ -21,6 +21,20 @@ <h3>Register</h3>
...
+				<label for="registerUsername">Username</label>
+				<input
+					name="username"
+					type="text"
+					class="form-control"
+					id="registerUsername"
+					placeholder="Enter your username" />
+			</div>
+		{% else %}
+			<!-- username is not loginAttribute. -->
```
Comment (LukeTowers) [inline components/account/register.htm]: You don't need this else statement, just the if is fine

**Comment by LukeTowers** [comment]
@JBonnyDev spaces not tabs for indentation please :)

**Comment by JBonnyDev** [comment]
@LukeTowers Hopefully up to spec now ^^

**Comment by LukeTowers** [comment]
Awesome, thanks @JBonnyDev!

**Pull request merged**